src/ctia/store_service_core.clj

```diff
@@ -0,0 +1,47 @@
+(ns ctia.store-service-core
+  (:require [clojure.string :as str]
+            [ctia.properties :as p]
+            [ctia.store :refer [empty-stores]]
+            [ctia.stores.es.init :as es-init]))
+
```

Comment:
These functions are from `ctia.init` and `ctia.store`, I left comments there as pointers.